app/src/main/java/org/fossasia/openevent/general/auth/LoginViewModel.kt

```diff
@@ -31,8 +33,8 @@ class LoginViewModel(
     val error: SingleLiveEvent<String> = mutableError
     private val mutableShowNoInternetDialog = MutableLiveData<Boolean>()
     val showNoInternetDialog: LiveData<Boolean> = mutableShowNoInternetDialog
-    private val mutableRequestTokenSuccess = MutableLiveData<Boolean>()
-    val requestTokenSuccess: LiveData<Boolean> = mutableRequestTokenSuccess
+    private val mutableRequestTokenSuccess = MutableLiveData<Pair<Boolean, String>>()
```

Comment:
Don't use pair. Create a local data class or deconstruct into multiple livedata (not recommended)

Reply:
Okay